test/helpers/mock_coordinator.cr

```diff
@@ -0,0 +1,16 @@
+class TestableCoordinator < Mosquito::Runners::Coordinator
+  # def only_if_coordinator : Nil
+  #   if @always_coordinator
+  #     yield
+  #   else
+  #     super
+  #   end
+  # end
+
+  # def always_coordinator!(always = true)
+  #   @always_coordinator = always
+  # end
```

Comment:
todo: figure out why this is crashing the compiler with a random error message

Reply:
What's the error?

Reply:
```text
In test/mosquito/runners/overseer_test.cr:36:10
Error: instantiating 'MockOverseer#run()'
In src/mosquito/runners/overseer.cr:46:9
In src/mosquito/runners/overseer.cr:53:20
In src/mosquito/runners/overseer.cr:53:20
In src/mosquito/runners/overseer.cr:55:21
In src/mosquito/runners/coordinator.cr:19:7
In test/helpers/mock_coordinator.cr:6:7

 6 | super
     ^----
Error: undefined method 'only_if_coordinator' for Colorize::ObjectExtensions
```

It's a bit far fetched, I have not had any time to dig into why it's throwing an undefined method error nor why it thinks it's limited to the wrong class.

Reply:
Evidently, this is because of https://github.com/crystal-lang/crystal/issues/10399